src/store/ChartSettingStore.js

```diff
@@ -9,6 +9,7 @@ export default class ChartSettingStore {
         this.defaultLanguage = this.languages[0];
         this.mainStore = mainStore;
         this.menu = new MenuStore(mainStore);
+        this.menu.setTag('setting');
```

Comment:
nitpick: set tag on constructor & remove the setTag() function.

`this.menu = new MenuStore(mainStore, {tag: 'settings-dialog'})`